packages/web3-net/src/index.ts

```diff
@@ -0,0 +1,18 @@
+import { Web3Context } from 'web3-core';
+import { EthExecutionAPI } from 'web3-common';
+
+import * as rpcMethodsWrappers from './rpc_method_wrappers';
+
+export default class Web3Net extends Web3Context<EthExecutionAPI> {
```

Comment:
I prefer to export such class as both class and default. That's more flexible way to cover different scenarios. 

first export class without default. 

```suggestion
export class Web3Net extends Web3Context<EthExecutionAPI> {
```

At the end of file export it as default. 

```ts
export default Web3Net;
```